### DISCUSSION TITLE
Add 'New contributor' as a search filter

### DISCUSSION THREAD

**Discussion by RCheesley** [community/community#19374] [Pull Requests]
When we have a PR from a new contributor, we want to make sure that we are prioritising them in our triage and review.

A nice way to help with that would be if it were possible to have a search flag that would highlight any PR from a new contributor, denoted thus:

![change allowRestart column as bool by avikarshasaha · Pull Request #11279 · mautic_mautic](https://user-images.githubusercontent.com/2930593/175608183-43fec174-3e71-4e8c-8d51-50f432b01086.png)

I tried searching the docs and I can't find that this is surfaced anywhere through the search filters but I think that it would be a really valuable aid to improving the contributor experience, and could open up some nice possibilities.

Hopefully as the information is already there, it should not be too much of a mission to implement!

**Comment by thuongtruong1009** [comment]
Hi @RCheesley 
As you can see, the first-time contributor badge is only for marking people who first create a new PR on your repository.

To engage people, while improving the contributing experience, and possibly opening up some nice possibilities. I think you should try using Github's action workflow feature.
* First you will create a greeting.yml file like [this link](https://github.com/thuongtruong1009/Snake-Game-OOP/blob/main/.github/workflows/greetings.yml)
* Edit the welcome bridge content to your liking
* Next, you put this file in folder name .github/workflow
-> Thus, every time someone new contributes to your repository for the first time, a new email will be sent to them. That will encourage users to increase their contributions to your repository and as a thank you to them.

Another, you can use [create issue branch tool](https://github.com/marketplace/create-issue-branch) for automation workflow

Hope the above sharing will help you. Thank you very much!🥰

**Comment by RCheesley** [comment]
Thanks for the ideas - we are already doing things like that (but not with the greeting.yml - thanks for the tip!), but we want to be able to have the ability to see for example PRs going inactive which are from new contributors, filter new bug fixes or features which are from new contributors and so forth.

So it's not really about meeting and greeting, more about triage and workflows.

**Comment by thuongtruong1009** [comment]
Hi @RCheesley 
You can try with colaborate feature by inviting some contributors to cooperate in your repository
* You click setting -> collaborator tab -> fill username of your partner
* if your partner accepts, in the future. Every contribution will be checked by them and a cooperation badge is also displayed on their PR
* Hope to useful for you!

**Comment by RCheesley** [comment]
@thuongtruong1009 thank you for replying, but this is not at all relevant to my discussion.
